src/plugins/guided_onboarding/public/components/guide_panel.tsx

```diff
@@ -203,46 +210,61 @@ export const GuidePanel = ({ api, application }: GuidePanelProps) => {
-                  defaultMessage: '{stepCount} steps',
-                  values: {
-                    stepCount: `${currentStep ? currentStep - 1 : 0} / ${guideConfig.steps.length}`,
-                  },
-                })}
-                max={guideConfig.steps.length}
-                size="l"
-              />
-
-              <EuiSpacer size="s" />
+              {/* Progress bar should only show after the first step has been complete */}
+              {stepsCompleted > 0 && (
+                <>
+                  <EuiSpacer size="xl" />
+                  <EuiProgress
+                    data-test-subj="guideProgress"
+                    label={i18n.translate('guidedOnboarding.dropdownPanel.progressLabel', {
+                      defaultMessage: 'Progress',
+                    })}
+                    value={stepsCompleted}
+                    valueText={i18n.translate('guidedOnboarding.dropdownPanel.progressValueLabel', {
+                      defaultMessage: '{stepCount} steps',
+                      values: {
+                        stepCount: `${stepsCompleted} / ${guideConfig.steps.length}`,
+                      },
+                    })}
+                    max={guideConfig.steps.length}
+                    size="l"
+                  />
+
+                  <EuiSpacer size="s" />
+                </>
+              )}
 
               <EuiHorizontalRule />
 
               {guideConfig?.steps.map((step, index, steps) => {
                 const accordionId = htmlIdGenerator(`accordion${index}`)();
-                const stepStatus = getStepStatus(steps, index, guideState?.activeStep);
-
-                return (
-                  <GuideStep
-                    accordionId={accordionId}
-                    stepStatus={stepStatus}
-                    stepConfig={step}
-                    stepNumber={index + 1}
-                    navigateToStep={navigateToStep}
-                    key={accordionId}
-                  />
-                );
+                const stepState = guideState?.steps[index];
+
+                if (stepState) {
+                  return (
+                    <GuideStep
+                      accordionId={accordionId}
+                      stepStatus={stepState.status}
+                      stepConfig={step}
+                      stepNumber={index + 1}
+                      navigateToStep={navigateToStep}
+                      key={accordionId}
+                    />
+                  );
+                }
               })}
+
+              {guideState?.status === 'ready_to_complete' && (
+                <EuiFlexGroup justifyContent="flexEnd">
+                  <EuiFlexItem grow={false}>
+                    <EuiButton onClick={completeGuide} fill data-test-subj="useElasticButton">
+                      {i18n.translate('guidedOnboarding.dropdownPanel.elasticButtonLabel', {
+                        defaultMessage: 'Continue using Elastic',
```

Comment:
We will probably add a config for the guide completion, I think Security has a redirect to a specific page at the end. Adding a note to https://github.com/elastic/kibana/issues/140976 so we don't forget.

Reply:
Now that I think about it, I thought we decided clicking this button would close the panel and keep them wherever they are, but let's confirm when we start on https://github.com/elastic/kibana/issues/140976 👍